lib/audited/auditor.rb

```diff
@@ -141,15 +139,11 @@ def audited_attributes
       protected
 
       def revision_with(attributes)
+        instance_variables = { "new_record" => self.destroyed?, "persisted" => !self.destroyed?, "readonly" => false, "destroyed" => false, "_destroyed" => false, "marked_for_destruction" => false }
```

Comment:
Not crazy about long-line hashes. This should be wrapped after each key/value pair, or else extracted into a method of its own.